chess/4-database/database.md

```diff
@@ -114,19 +114,70 @@ As part of your unit test deliverable you need to meet the following requirement
 
 For this phase the TAs will grade the quality of your project's source code. The rubric used to evaluate code quality can be found here: [Rubric](../code-quality-rubric.md)
 
+### Common Submission Issues
+
+Since you're adding code that acts with an external program in Phase 4, the likelihood of having auto-grader test
+output different from your local output is higher than previous phases.
+
+If you encounter this, here are some common things to try (expand the collapsible section).
+
+<details>
+
+<summary>Tips for resolving AutoGrader submission issues when local tests pass</summary>
+
+> [!IMPORTANT]
+> This guide assumes that your tests **are passing locally**.\
+> If your tests are not passing on your own computer,\
```

Comment:
Why the line breaks?

Reply:
I personally like to have short sentences on their own line to emphasize their importance. I like to control the way the text wraps so that each sentence is squarely on it's own line and the note forms a rectangular block rather than a wide line.

This is the writer in my coming out to express a preference about the way the text displays on the screen (something Steve Jobs wouldn't complain about). However, I am in now way going to say that my preferences are the best ones. I'm not claiming to be a UI designer, but I can do my best.

This is a detail that can be easily accommodated if you prefer that I don't exert control on the line wraps. Just reply back if you want them gone.